CoWorkMobileApp/Controllers/CollectionsViewController.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CollectionsViewController.swift
+//  CoWorkMobileApp
+//
+//  Created by Qadriyyah Thomas on 8/10/22.
+//
+
+import UIKit
+
+class CollectionsViewController: UIViewController {
```

Comment:
Why did you need to do this?

Reply:
This is the view controller for the user's saved workspaces. Maybe I should rename it.